edenai_apis/apis/neuralspace/neuralspace_api.py

```diff
@@ -204,14 +206,19 @@ def audio__speech_to_text_async__launch_job(
         payload.update(provider_params)
 
         response = requests.post(url=url_file_transcribe, headers=headers, data=payload)
-        original_response = response.json()
+
         if response.status_code != 201:
-            raise ProviderException(
-                original_response.get("data").get("error"), code=response.status_code
-            )
+            status = response.status_code
+            message = response.text
```

Comment:
Here message will always have as a value either: ```extract(data, ["data", "error"], fallback=message) ```  or ```HTTPStatus(status).phrase```, so maybe initialize it with either of these values ?